server/api/categoryRouter.js

```diff
@@ -36,14 +37,18 @@ router.post('/createCategory', (req, res) => {
  * 
  * response: [{ title: String, _id: Mongoose.objectId}]
  */
-router.get('/getCategories', async (req, res) => {
-  const userId = req.body.userId;
+router.get('/getCategories', async (req, res, next) => {
   try {
-    const categoriesData = await Category.find({ userId: userId }, 'title', (error, data)).exec(); 
+    const userId = req.body.userId;
+    const categoriesData = await Category.find({ userId: userId }, 'title', function (err, data) {
+      if (err) {
+        next(err);
+      }
+    });
```

Comment:
My bad for not deleting (error, data) on the original line 42, will do that. 

But for lines 44-46: I thought using await allows you to not do error handling through the callback, and the try block will just catch it. [https://stackoverflow.com/questions/48601249/how-to-return-mongoose-query-results-without-callback](url)

Reply:
so technically the callback and error handling in it shouldn't be needed?

Reply:
Yes, you're correct! Awesome catch